report_batch/models/ir_action_report.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (C) 2019 IBM Corp.
+# Copyright (C) 2019 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+import random
+
+
+class IrActionsReportSubreport(models.Model):
+    _name = 'ir.actions.report.subreport'
+    _description = 'Report Subreport'
+    _order = 'sequence'
+
+    parent_report_id = fields.Many2one('ir.actions.report', ondelete='cascade')
+    sequence = fields.Integer(default=10)
+    model = fields.Char(related='parent_report_id.model')
+    subreport_id = fields.Many2one(
+        'ir.actions.report', string='Subreport', required=True)
+
+
+class IrActionsReport(models.Model):
+    _inherit = 'ir.actions.report'
+
+    subreport_ids = fields.One2many(
+        'ir.actions.report.subreport', 'parent_report_id')
+    model_id = fields.Many2one('ir.model', string='Model')
+
+    @api.onchange('model_id')
+    def onchange_model_id(self):
+        if self.model_id:
+            self.model = self.model_id.model
```

Comment:
Isn't `model_id` a computed field?
`model` is the editable one.

Reply:
same as above.